src/main/webapp/WEB-INF/assets-manifest.json

```diff
@@ -32,8 +32,7 @@
   "npm.chosen-js.css": "/assets/npm.chosen-js.12c12173cb2425ad9684.css",
   "npm.chosen-js.js": "/assets/npm.chosen-js.476270707b7692a11c69.min.js",
   "npm.core-js.js": "/assets/npm.core-js.34519420b2a228bfccb8.min.js",
-  "npm.date-fns.de.js": "/assets/npm.date-fns.de.b7e02b76f6a1f4670ba8.min.js",
```

Comment:
@bseber weisst du warum das rausfliegt?

Reply:
in unserer [webpack config](https://github.com/synyx/urlaubsverwaltung/blob/8faf6439d2ed9298c5ba30cf70b6d9659d7bccb3/webpack.config.js#L142-L151) wird nach dem pfad geschaut und dann ein extra asset angelegt. `date-fns` hat die struktur geändert und unser check passt daher nicht mehr. (alt: `date-fns/locale` neu: `date-fns/esm/locale`)

Reply:
danke dir, kannst du bitte einmal ueber den PR drueber gehen und schauen ob das so gut aussieht und dann mergen wenn ja?

Reply:
k, der Fehler ist für mich derzeit wirr. Kannst du da bei Gelegenheit mal drauf schauen?